package.json

```diff
@@ -30,20 +36,15 @@
     "mongodb": "^3.5.5",
     "prettier": "1.15.3",
     "pretty-quick": "^1.11.1",
+    "rollup": "^2.7.2",
     "ts-loader": "^6.2.1",
-    "typescript": "^3.7.5",
-    "webpack": "^4.41.5",
-    "webpack-cli": "^3.3.10"
+    "typescript": "^3.8.3"
   },
   "main": "./index.js",
-  "module": "./es5m/index.js",
```

Comment:
I'd like to keep `modules` and `es2015` in place. At least until ES modules are not mature enough (let's wait for Node 14 LTS)